javascripts/discourse/components/welcome-link-banner.hbs

```diff
@@ -0,0 +1,33 @@
+{{#unless this.dismissedBanner}}
+  {{#if this.shouldShow}}
```

Comment:
I'd add `this.dismissedBanner` (btw. `this.dismissed` as a name would also work 😉) to `shouldShow` getter then this would be a single `{{#if}}` block